electrophysiology/src/CellModel/OHaraRudyIsoParameters.cpp

```diff
@@ -0,0 +1,384 @@
+  P[VT_A_h_fast].value = 0.99;
+  P[VT_A_h_slow].value = 0.01;
+  P[VT_A_f_fast].value = 0.6;
+  P[VT_k_Na1].value    = 15.0;
+  P[VT_k_Na2].value    = 5.0;
+  P[VT_k_asymm].value  = 12.5;
+  P[VT_k_Ca_on].value  = 1.5e6;
+  P[VT_k_Ca_off].value = 5.0e3;
+  P[VT_k_m_1].value    = 182.4;
+  P[VT_k_p_2].value    = 687.2;
+  P[VT_k_p_4].value    = 639.0;
+  P[VT_MgADP].value    = 0.05;
+  P[VT_MgATP].value    = 9.8;
+  P[VT_K_MgATP].value  = 1.698e-7;
+  P[VT_beta_tau].value = 4.75;
+  P[VT_P_Ca].value     = P[VT_PCa_Multiplier].value * 0.0001;
+
+  /// cell geometry
+  P[VT_v_cell].value =  1000.0 * 3.14 * P[VT_r].value * P[VT_r].value * P[VT_L].value;
+  P[VT_A_geo].value  =  2.0 * 3.14 * P[VT_r].value * P[VT_r].value + 2.0 * 3.14 * P[VT_r].value * P[VT_L].value;
+  P[VT_A_cap].value  =  2.0 * P[VT_A_geo].value;
+  P[VT_v_myo].value  =  0.68 * P[VT_v_cell].value;
+  P[VT_v_nsr].value  =  0.0552 * P[VT_v_cell].value;
+  P[VT_v_jsr].value  =  0.0048 * P[VT_v_cell].value;
+  P[VT_v_ss].value   =  0.02 * P[VT_v_cell].value;
+
+  /// permeability to ion X
+  P[VT_P_CaNa].value      = 0.00125 * P[VT_P_Ca].value;
+  P[VT_P_CaK].value       = 3.574e-04 * P[VT_P_Ca].value;
+  P[VT_P_Ca_CaMK].value   = 1.1 * P[VT_P_Ca].value;
+  P[VT_P_CaNa_CaMK].value = 0.00125 * P[VT_P_Ca_CaMK].value;
+  P[VT_P_CaK_CaMK].value  = 3.5743e-04 * P[VT_P_Ca_CaMK].value;
+  P[VT_P_Ca_PKA].value    = 1.9 * P[VT_P_Ca].value;
+  P[VT_P_CaNa_PKA].value  = 0.00125 * P[VT_P_Ca_PKA].value;
+  P[VT_P_CaK_PKA].value   = 3.5743e-04 * P[VT_P_Ca_PKA].value;
+
+  /// fraction of channels with gate X undergoing fast/slow process
+  P[VT_A_f_slow].value = 1.0 - P[VT_A_f_fast].value;
+
+  /// ??
+  P[VT_h_10].value = P[VT_k_asymm].value + 1.0 + (P[VT_Na_o].value * (1.0 + P[VT_Na_o].value / P[VT_k_Na2].value)) /
+    P[VT_k_Na1].value;
+  P[VT_h_11].value = (P[VT_Na_o].value * P[VT_Na_o].value) / (P[VT_h_10].value * P[VT_k_Na1].value * P[VT_k_Na2].value);
+  P[VT_h_12].value = 1.0 / P[VT_h_10].value;
+  P[VT_k_1].value  = P[VT_h_12].value * P[VT_Ca_o].value * P[VT_k_Ca_on].value;
+  P[VT_k_2].value  = P[VT_k_Ca_off].value;
+  P[VT_k_5].value  = P[VT_k_Ca_off].value;
+
+  // (de)phosphorylation rates
+  P[VT_beta_1].value  = P[VT_k_m_1].value * P[VT_MgADP].value;
+  P[VT_alpha_2].value = P[VT_k_p_2].value;
+  P[VT_alpha_4].value = (P[VT_k_p_4].value * (P[VT_MgATP].value / P[VT_K_MgATP].value)) /
+    (1.0 + (P[VT_MgATP].value / P[VT_K_MgATP].value));
+  P[VT_alpha_rel].value      = 0.5 * P[VT_beta_tau].value;
+  P[VT_beta_tau_CaMK].value  = 1.25 * P[VT_beta_tau].value;
+  P[VT_alpha_rel_CaMK].value = 0.5 * P[VT_beta_tau_CaMK].value;
+  P[VT_alpha_rel_PKA].value = 2.5 * P[VT_alpha_rel].value;
+  P[VT_alpha_rel_both].value = 2.5 * P[VT_alpha_rel_CaMK].value;
+
+  // time constant of gate X
+  P[VT_tau_h_LCaMK].value = 3.0 * P[VT_tau_h_L].value;
+
+  P[VT_RToverF].value = (P[VT_R].value * P[VT_T].value) / P[VT_F].value;
+    
+  // PKA fraction
+  //P[VT_ICaL_P_frac].value  = 0.987823373540079;
```

Comment:
Please remove unused commented parameters or add a comment what they are/why you want to keep them here